src/charts/axis.js

```diff
@@ -154,10 +154,17 @@ export default function Axis (_container) {
     if (config.yAxisPercentageFormat) {
       axis.tickFormat(formatPercentage(config.yAxisPercentageFormat))
     } else if (typeof config.yAxisFormat === "function") {
-      const measureName = scales.measureNameLookup("y")
-      const hasFormatterForMeasure = config.yAxisFormat(null, measureName)
+
+      const axisKey = scales.hasSecondAxis
+        ? "y2"
+        : "y"
+      const yAxisFormatFunction = scales.hasSecondAxis
```

Comment:
For symbol-aware tooling and type safety (if/when we do introduce typing here), it might be better to do this with the function values directly instead of the keys: 

```
const yAxisFormatFunction = scales.hasSecondAxis
        ? config.y2AxisFormat
        : config.yAxisFormat
```

Reply:
You are absolutely right about that. Good catch! Changed!